package-lock.json

```diff
@@ -5934,11 +5934,6 @@
         "node": ">= 8.0.0"
       }
     },
-    "node_modules/express-validator/node_modules/lodash": {
```

Comment:
`express-validator` doesn't need `lodash` anymore? Or it will use the one we install directly?

Reply:
I believe it is using the one installed directly now. Not sure why it wasn't using it before, as the version numbers were precisely the same as they are now.

I guess `npm audit` must have simplified the dependency tree in the process.